src/Miningcore/Persistence/Model/TransactionReceipt.cs

```diff
@@ -0,0 +1,23 @@
+using Nethereum.Web3;
+
+namespace Miningcore.Persistence.Model
+{
+    public class TransactionReceipt
+    {
+        public TransactionReceipt(Nethereum.RPC.Eth.DTOs.TransactionReceipt tx)
+        {
+            TransactionHash = tx.TransactionHash;
+            EffectiveGasPrice = Web3.Convert.FromWei(tx.EffectiveGasPrice);
+            GasUsed = Web3.Convert.FromWei(tx.GasUsed);
+        }
+
+        public TransactionReceipt(string id)
+        {
+            TransactionHash = id;
+        }
+
+        public string TransactionHash { get; set; }
+        public decimal GasUsed { get; set; }
```

Comment:
since Gas is specific to Ethereum, we're going to need to enhance this when we move to another coid.  is that acceptable?

Reply:
Better I'll change names generic for now. Also, it can be updated when we have our next coin in place.